src/main/java/uk/ac/sanger/sccp/stan/service/register/filereader/SectionRegisterFileReader.java

```diff
@@ -19,6 +19,7 @@ enum Column implements IColumn {
         Work_number(Pattern.compile("(work|sgp)\\s*number", Pattern.CASE_INSENSITIVE)),
         Slide_type,
         External_slide_ID,
+        Prebarcode(String.class, Pattern.compile("(xenium( slide)?\\s*|pre)barcode", Pattern.CASE_INSENSITIVE), false),
```

Comment:
Is this space intended - xenium( slide)  or is it xenium(slide)

Reply:
It's intended so it will match the heading "xenium slide barcode" or "xenium barcode" or "prebarcode"